nbrowser

```diff
@@ -152,7 +152,7 @@ open_a_browser(){
 			;;
 		*)
 			selected_browser=$(printf '%s' "${selected_browser}" | awk '{split($0,a,": "); print a[2]}')
-			{ setsid -f ${selected_browser} >/dev/null 2>&1 ; }
+			{ setsid -f $(bash -c "exec ${selected_browser} \"$*\"") >/dev/null 2>&1 ; }
```

Comment:
no need to pass args
```
{ setsid -f $(bash -c "exec ${selected_browser}") >/dev/null 2>&1 ; }
```

Reply:
Thanks. That was a blind copy/paste mistake. Fixed.

Reply:
Thanks. That was a blind copy/paste mistake. Fixed.